domain/src/user.rs

```diff
@@ -1,11 +1,16 @@
-use crate::{error::Error, users, Id};
-use entity_api::{mutate, query, query::IntoQueryFilterMap};
-use sea_orm::DatabaseConnection;
-use sea_orm::IntoActiveModel;
-
+use crate::{
+    error::Error,
+    error::{DomainErrorKind, EntityErrorKind, InternalErrorKind},
+    users, Id,
+};
+use chrono::Utc;
 pub use entity_api::user::{
-    create, find_by_email, find_by_id, find_by_organization, AuthSession, Backend, Credentials,
+    create, create_by_organization, find_by_email, find_by_id, find_by_organization, AuthSession,
```

Comment:
`create_by_organization` will be the function we use once we decouple the user creation from coaching relationship assignment